src/block/block_map.c

```diff
@@ -58,12 +50,15 @@ __wt_block_map(
 
 	/*
 	 * Map the file into memory.
-	 * Ignore errors, we'll read the file through the cache if map fails.
+	 * Ignore not-supported errors, we'll read the file through the cache
+	 * if map fails.
 	 */
-	(void)__wt_mmap(session, block->fh, mapp, maplenp, mappingcookie);
+	ret = block->fh->fh_map(session, block->fh, mapp, maplenp);
+	if (ret == ENOTSUP)
```

Comment:
I don't believe checking ENOTSUP is safe across platforms. We don't translate error codes from OS calls to be POSIX compliant.